docs/spec/v1alpha1/imagepolicies.md

```diff
@@ -124,5 +124,73 @@ The `LatestImage` field contains the image selected by the policy rule, when it
 There is one condition that may be present: the GitOps toolkit-standard `ReadyCondition`. This will
 be marked as true when the policy rule has selected an image.
 
-[image-automation-controller]: https://github.com/image-automation-controller
-[semver-range]: https://github.com/Masterminds/semver#checking-version-constraints
+## Examples
```

Comment:
It might make sense to send people to the [regex syntax doc page](https://golang.org/pkg/regexp/syntax/) for help with compiling supported regular expression patterns.
In the future, once we can develop validation webhooks, we'll be able to provide feedback to the user if the regular expression is incorrect.

Reply:
I've added the link to Go docs. Thanks @relu for the suggestion.